src/sql/workbench/browser/modelComponents/media/card.css

```diff
@@ -221,3 +221,38 @@
 	text-align: right;
 	white-space: nowrap;
 }
+
+
+.model-left-icon-card {
```

Comment:
please add .horizontal before all of these, the styles are not necessarily translatable between horizontal and vertical. See above in the CSS file for examples